dataeng/jobs/analytics/SnowflakeSchemaBuilder.groovy

```diff
@@ -14,6 +14,7 @@ class SnowflakeSchemaBuilder {
                 stringParam('WAREHOUSE_TRANSFORMS_URL', allVars.get('WAREHOUSE_TRANSFORMS_URL'), 'URL for the Warehouse Transforms Repo.')
                 stringParam('WAREHOUSE_TRANSFORMS_BRANCH', allVars.get('WAREHOUSE_TRANSFORMS_BRANCH'), 'Branch of Warehouse Transforms to use.')
                 stringParam('SCHEMAS', allVars.get('SCHEMAS'), 'Snowflake schemas to work on')
+                stringParam('RAW_SUFFIX', allVars.get('RAW_SUFFIX'), 'List of suffixes to remove from Schema name')
```

Comment:
Can we change this to `RAW_SUFFIXES` to match the script, and make it more obvious you can give more than one?

Reply:
Done